middleware/middleware-1.3.23.md

```diff
@@ -0,0 +1,68 @@
+---
+slug: /release-notes/middleware/1.3.23
+title: Version 1.3.23
+---
+
+# fiskaltrust.Middleware 1.3.23 (Germany)
+_September 15, 2021_
+
+In this version of the Middleware, we added the possibility to easily identify mandatory and optional signature items when printing receipts. Additionally, the Middleware is now able to automatically update the _Fiskal Cloud Connector_, and several bugs and stability issues have been resolved.
+
+
+:::caution
+
+Version 1.3 of the Middleware is meant for the German market only, customers in Austria and France should continue to use version 1.2. We will unify these experiences in an upcoming version.
+
+:::
```

Comment:
Anything missing?

Reply:
I don't think so, this ::: are automatically converted by Docusaurus (see above)